pizza-order-showcase/pom.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.miragon.miranum.consulting</groupId>
+    <artifactId>miranum-pizza-order</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <name>miranum-pizza-order</name>
+    <packaging>pom</packaging>
+    <description>Miranum pizza order example</description>
+
+    <properties>
+        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <java.version>17</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <spring.boot.version>2.7.8</spring.boot.version>
+        <camunda7.version>7.18.0</camunda7.version>
+        <camunda8.version>8.1.4</camunda8.version>
+        <spring.cloud.version>2021.0.3</spring.cloud.version>
+        <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
+        <lombok.version>1.18.26</lombok.version>
+        <lombok.mapstruct.binding.version>0.2.0</lombok.mapstruct.binding.version>
+        <org.mapstruct.version>1.5.3.Final</org.mapstruct.version>
+        <maven.surfire.plugin.version>2.22.2</maven.surfire.plugin.version>
+        <okhttp.version>4.8.1</okhttp.version>
+        <jacoco.version>0.8.8</jacoco.version>
+    </properties>
+
+    <modules>
+        <module>waiter</module>
+        <module>kitchen</module>
+        <module>notification</module>
+        <module>pizza-order-frontend</module>
+    </modules>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring.boot.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <!-- Import dependency management from Spring Cloud -->
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-dependencies</artifactId>
+                <version>${spring.cloud.version}</version>
```

Comment:
I guess we can clean up the pom a bit. For example we don't need spring-cloud-deps